Dynatrace/dynatrace-operator
# Streamline init container config

## Description

We should only have 1 way to configure each thing, so the same thing can't be in env and the secret.
The general rule is that the env should be used for things that are only available at the time of injection.

What was moved?
- InstallerMode
- CSIMode
- ReadOnlyCSI

## How can this be tested?

Everything should work as before, deploy a Dynakube with applicationMonitoring and test -> readOnly CSI, set a fixed version and a dynakube without readOnlyCSI but CSI still enabled

## Checklist
